src/test/java/com/bhoruka/bloodbank/TestConstants.java

```diff
@@ -0,0 +1,45 @@
+package com.bhoruka.bloodbank;
+
+import com.bhoruka.bloodbank.model.CampModel;
+import com.bhoruka.bloodbank.model.request.CreateCampRequest;
+import com.bhoruka.bloodbank.model.response.CreateCampResponse;
+
+import java.util.Date;
+
+import org.springframework.http.HttpStatus;
+
+public final class TestConstants {
```

Comment:
Each manager will have its own set of test constants. Would it be an overkill if we decided to separate out TestConstants for each, and this would contain only the common ones (if any)?

Reply:
Sure. Will rename this file to TestCampConstants.